backend/controllers/attendance/offenderActivities.js

```diff
@@ -1,6 +1,6 @@
 const { switchDateFormat } = require('../../utils')
 
-const offenderActivitesFactory = (elite2api, whereaboutsApi) => {
+const offenderActivitesFactory = (prisonapi, whereaboutsApi) => {
```

Comment:
Camel case here would be preferable.

Reply:
sure sounds good, does this apply only here or throughout all the files?

Reply:
For consistency, I think throughout, but it's right in most of the places already :)